app/src/main/java/com/x8bit/bitwarden/ui/vault/feature/addedit/VaultAddEditSshKeyItems.kt

```diff
@@ -0,0 +1,127 @@
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.x8bit.bitwarden.R
+import com.x8bit.bitwarden.ui.platform.components.field.BitwardenPasswordField
+import com.x8bit.bitwarden.ui.platform.components.field.BitwardenTextField
+import com.x8bit.bitwarden.ui.platform.theme.BitwardenTheme
+import com.x8bit.bitwarden.ui.vault.feature.addedit.handlers.VaultAddEditCommonHandlers
+import com.x8bit.bitwarden.ui.vault.feature.addedit.handlers.VaultAddEditSshKeyTypeHandlers
+
+/**
+ * The UI for adding and editing a SSH key cipher.
+ */
+fun LazyListScope.vaultAddEditSshKeyItems(
+    commonState: VaultAddEditState.ViewState.Content.Common,
+    sshKeyState: VaultAddEditState.ViewState.Content.ItemType.SshKey,
+    commonTypeHandlers: VaultAddEditCommonHandlers,
+    sshKeyTypeHandlers: VaultAddEditSshKeyTypeHandlers,
+) {
+    item {
+        Spacer(modifier = Modifier.height(8.dp))
+        BitwardenTextField(
+            label = stringResource(id = R.string.name),
+            value = commonState.name,
+            onValueChange = commonTypeHandlers.onNameTextChange,
+            modifier = Modifier
+                .testTag("ItemNameEntry")
+                .fillMaxWidth()
+                .padding(horizontal = 16.dp),
+        )
+    }
+
+    item {
+        Spacer(modifier = Modifier.height(8.dp))
+        BitwardenTextField(
+            label = stringResource(id = R.string.public_key),
+            value = sshKeyState.publicKey,
+            onValueChange = sshKeyTypeHandlers.onPublicKeyTextChange,
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(horizontal = 16.dp),
+        )
+    }
+
+    item {
+        Spacer(modifier = Modifier.height(8.dp))
+        BitwardenPasswordField(
+            label = stringResource(id = R.string.private_key),
+            value = sshKeyState.privateKey,
+            onValueChange = sshKeyTypeHandlers.onPrivateKeyTextChange,
+            showPassword = sshKeyState.showPrivateKey,
+            showPasswordChange = { sshKeyTypeHandlers.onPrivateKeyVisibilityChange(it) },
+            showPasswordTestTag = "ViewPrivateKeyButton",
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(horizontal = 16.dp),
+        )
+    }
+
+    item {
+        Spacer(modifier = Modifier.height(8.dp))
+        BitwardenTextField(
+            label = stringResource(id = R.string.fingerprint),
+            value = sshKeyState.fingerprint,
+            onValueChange = sshKeyTypeHandlers.onFingerprintTextChange,
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(horizontal = 16.dp),
+        )
+    }
+}
+
+@Preview(showBackground = true)
+@Composable
+private fun VaultAddEditSshKeyItems_preview() {
+    var showPublicKey = remember { mutableStateOf(false) }
```

Comment:
Do we need this?

Reply:
Nope. Forgot to remove it while I was playing with the preview. Thanks catching it. 😅